Add docs on multiline comments (#893)
neo4j/docs-cypher: modules/ROOT/pages/syntax/comments.adoc

:description: This section describes how to use comments in Cypher.

[[cypher-comments]]
= Comments

This page describes how to use comments in Cypher.

A single line comment begins with double slash (`//`) and continues to the end of the line.

> A multi-line comment begins with a slash and asterisk (`/&#42;`) and continues until the closing of the comment, which is done with an asterisk and a slash ('&#42;/'
> )

Reviewer: ```suggestion
A multi-line comment begins with a slash and asterisk (`/&#42;`) and continues until it ends with an asterisk and a slash ('&#42;/').
```

RETURN n
----

[source, cypher, indent=0]
----
MATCH (n) /* This is a multi line comment,

> the comment is still going on this line as well
> but we are closing it on this line */

Reviewer: ```suggestion
the comment continues on this line
but it ends on this line. */
```
